docs/DeveloperGuide.md

```diff
@@ -607,8 +607,22 @@ testers are expected to do more *exploratory* testing.
 
 ### Saving data
 
-1. Dealing with missing/corrupted data files
+1. Dealing with missing files
 
-   1. _{explain how to simulate a missing/corrupted file, and the expected behavior}_
+   1. Delete `config.json` and re-launch the app. Expected: New `config.json` created. Exisitng data not affected.
 
-1. _{ more test cases …​ }_
+   2. Delete `preferences.json` and re-launch the app. Expected: New `preferences.json` created. Exisitng data not affected.
+
+   3. Edit the line `"addressBookFilePath" : "data/socialbook.json"` to `"addressBookFilePath" : "data/data.json"` and re-launch the app. Expected: App starts on clean slate (i.e. with sample data only).
+
+   4. Delete `data/` or `data/socialbook.json`. Expected:  New `data/socialbook.json` created. App starts on clean slate (i.e. with sample data only).
+
+2. Dealing with corrupted data files
+
+   1. Add `"p"` to a `"phone"` field in data file. Expected: The person with the `"p"` in `"phone"` field is lost. The rest of the contact still exists.
```

Comment:
For the multiple "rest of the contact still exists" below, is this a reference to the remaining contacts still existing? If so, then perhaps adding a word to make it "rest of the contact list still exists" could improve clarity

Reply:
right